### PR TITLE
Added field status

### LINKED ISSUE #14: Add documentation on which fields are currently supported / what are planned to be supported

### PR DESCRIPTION
**Pull request by noemide**
Added documentation on all fields that can possibly be synced, their expected source of truth and the implementation status (active/planned).
Closes #14

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,123 @@
...
+|head_title_after_name | Directory | TODO |
+|head_role | Directory | TODO |
+|contact | *see section contact* |
+|latitude | Directory | TODO |
+|longitude | Directory | TODO |
+|sex | FHIR | TODO |
+|diagnosis_available | FHIR | TODO |
+|age Low| FHIR | TODO |
+|age High| FHIR | TODO |
+|age Unit| FHIR | *Fixed to years* |
```
[the remark this answers is not in the document]
Comment (CeciliaEngels) [reply README.md:89]: @alexanderkiel , was sagt der LIFE Child-Daten Experte hierzu? *fixed to years*

**Review by CeciliaEngels — COMMENTED** [review]
Beautiful! Thank you 🥇

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (noemide) [reply README.md:89]: Nach Diskussion -> Nur Jahre

File: README.md [hunk 2]
```diff
@@ -2,6 +2,123 @@
...
+|head_title_before_name | Directory | TODO |
+|head_firstname | Directory | TODO |
+|head_lastname | Directory | TODO |
+|head_title_after_name | Directory | TODO |
+|head_role | Directory | TODO |
+|contact | *see section contact* |
+|latitude | Directory | TODO |
+|longitude | Directory | TODO |
+|sex | FHIR | TODO |
+|diagnosis_available | FHIR | TODO |
```
[the remark this answers is not in the document]
Comment (noemide) [reply README.md:86]: From Donor, and ALL THE DIAGNOSES! 🤞

**Pull request merged**